### PR TITLE
Fix issue #129 - PathParticle throws error when no speed interp

### PR DESCRIPTION
**Pull request by DustinWoods**

### CODE REVIEW

File: src/PathParticle.ts [hunk 1]
```diff
@@ -150,8 +150,13 @@ export class PathParticle extends Particle
 		if(lerp >= 0 && this.path)
 		{
 			//increase linear movement based on speed
-			const speed = this.speedList.interpolate(lerp) * this.speedMultiplier;
-			this.movement += speed * delta;
+			if(this._doSpeed) {
+				const speed = this.speedList.interpolate(lerp) * this.speedMultiplier;
+				this.movement += speed * delta;
+			} else {
+				const speed = ParticleUtils.length(this.velocity);
```
Comment (andrewstart) [inline src/PathParticle.ts:157]: Getting the length of the velocity is inefficient (`sqrt()` is involved) - I would go with `this.speedList.current.value * this.speedMultiplier`.

Comment (DustinWoods) [reply src/PathParticle.ts]: Good point! Got it updated.

**Comment by andrewstart** [comment]
Thanks, I have one other fix I need to put in, and then I'll make a release probably tonight or tomorrow when I get some time.

**Comment by DustinWoods** [comment]
Awesome, thanks for being so quick!

**Pull request merged**